addons/mail/static/src/new/core/persona_model.js

```diff
@@ -19,6 +19,8 @@ export class Persona {
     type;
     /** @type {string} */
     name;
+    /** @type {string} */
+    displayName;
```

Comment:
At some point, we may want to reimplement the computing logic of displayName in JavaScript rather than retrieving it from the server, but for now I guess it's okay to do it like this, it saves you having to change the python part 👍️